casts/addressText/src/UIAddressTextView.tsx

```diff
@@ -31,12 +31,16 @@ type UIAddressTextViewProps = UIMaterialTextViewProps & {
     };
 };
 
-function useAddressTextView(ref: React.Ref<TextInput> | null, props: UIAddressTextViewProps) {
+function useAddressTextView(props: UIAddressTextViewProps) {
+    /**
+     * ref is passed as null because types of ref are incompatible
+     * and we don't need the clear method
```

Comment:
Are you sure we don't need it? 🤔

Reply:
clear method returns from useUITextViewValue but it doesn't take here.